src/api/endpoint.ts

```diff
@@ -1,3 +1,6 @@
 import { serloDomain } from '@/helper/serlo-domain'
 
-export const endpoint = `https://api.${serloDomain}/graphql`
+export const endpoint =
+  process.env.NEXT_PUBLIC_ENV === 'local'
+    ? 'http://localhost:3001/graphql'
```

Comment:
We should permanently change up the port of API or frontend so that they don't overlap. Ignoring that for now.